design/Geth_type_conventions.md

```diff
@@ -0,0 +1,48 @@
+# Code Conventions Proposal - Geth Type Renames
+
+This proposal seeks to establish a standard on how the enclave codebase should use the standard `geth` types like `types.Transaction`, `types.Block` and the likes.
+Currently the code base sparsely uses renamed types like the ones defined in `common/types.go`. The suggestion of this proposal is that such type renames should be the standard
```

Comment:
maybe use "aliased" instead of rename. 
And also show an example so people understand what you mean